README.md

```diff
@@ -4,7 +4,7 @@
 
 New Relic's official Apollo Server plugin for use with the [Node.js agent](https://github.com/newrelic/node-newrelic).
 
```

Comment:
Should we be more explicit about how the graphql server needs to have the agent required?

Reply:
Wouldn't hurt